nixos/modules/services/misc/jellyfin.nix

```diff
@@ -1,33 +1,70 @@
 { config, pkgs, lib, ... }:
 
-with lib;
-
 let
+  inherit (lib.modules) mkIf;
+  inherit (lib.meta) getExe maintainers;
+  inherit (lib.options) mkEnableOption mkOption mkPackageOption mdDoc;
+  inherit (lib.types) str path bool;
```

Comment:
Where the functions are exact under lib is an implementation detail, we can just take them all from lib itself.

Reply:
moving to one line? I've done so now.

Reply:
@SuperSandro2000 disagree, there are collisions, and in general being explicit is preferable. It's also in line with other modules e.g. https://github.com/NixOS/nixpkgs/pull/267900/files

Reply:
id prefer being more explicit and inline with other modules mentioned within that issue but happy to do either. ill leave it as the two lines for now, one for (lib) and one for (lib.types)